lib/buttons-and-links/KButton.vue

```diff
@@ -13,23 +13,23 @@
   >
     <!-- icon may come by slot or by prop -->
     <slot name="icon"></slot>
-    <KIcon 
-      v-if="icon" 
-      :icon="icon" 
+    <KIcon
+      v-if="icon"
+      :icon="icon"
       :color="iconColor"
       class="prop-icon"
     />
 
     <slot v-if="$slots.default"></slot>
 
     <template v-else>
-      <span class="link-text">{{ text }}</span>
+      <span class="link-text" :style="spanStyle">{{ text }}</span>
```

Comment:
`spanStyle` is a big vague. Maybe something like `textStyle` would be a bit clearer?